README.md

```diff
@@ -24,9 +24,9 @@ released version of Riot:
 Releases are signed by PGP, and can be checked against the public key
 at https://riot.im/packages/keys/riot.asc
 
-Note that Chrome does not allow microphone or webcam access for sites served
-over http (except localhost), so for working VoIP you will need to serve Riot
```

Comment:
It'd be nice if the exception for localhost didn't disappear

Reply:
the last sentence actually mentions thus